xcode/Subconscious/Shared/Components/Detail/MemoEditorDetailMetaSheetView.swift

```diff
@@ -54,7 +54,26 @@ struct MemoEditorDetailMetaSheetView: View {
                             )
                         }
                         .buttonStyle(RowButtonStyle())
+                        
                         Divider()
+                        
+                        if let address = store.state.address {
+                            Button(
+                                action: {
+                                    store.send(.requestQuoteInNewNote(address))
+                                },
+                                label: {
+                                    Label(
+                                        "Quote in new note",
+                                        systemImage: "quote.opening"
```

Comment:
I welcome discussion on the best wording and icon for this action.

To me "linking" is inserting a slashlink into an existing note and "quoting" is creating a new note that references / reflects on another note (using linking).